config.xml

```diff
@@ -113,5 +113,16 @@
     </plugin>
     <plugin name="cordova-plugin-sms" spec="^1.0.5" />
     <plugin name="cordova-sms-plugin" spec="^0.1.11" />
+    <plugin name="cordova-plugin-buildinfo" spec="^2.0.1" />
+    <plugin name="cordova-plugin-browsertab" spec="^0.2.0" />
+    <plugin name="cordova-plugin-inappbrowser" spec="~1.7.2" />
+    <plugin name="cordova-universal-links-plugin" spec="^1.2.1" />
+    <universal-links>
+        <host name="j68c5.app.goo.gl/R1XM" scheme="https" />
+        <host name="universalgamemaker.firebaseapp.com" scheme="https">
+            <path url="/__/auth/callback" />
+        </host>
+    </universal-links>
     <engine name="android" spec="~7.0.0" />
+    <plugin name="com.dbaq.cordova.contactsPhoneNumbers" spec="~0.0.10" />
```

Comment:
If this plugin gives only thumbnails on android, you can remove it (we need both ios and android)